package.json

```diff
@@ -10,15 +10,17 @@
         "url": "github.com/jamestalmage"
     },
     "engines": {
-        "node": ">=8"
+        "node": "^14.18.0 || >=16.0.0"
     },
     "scripts": {
         "prepublishOnly": "npm run create-types",
-        "test": "xo && nyc ava",
-        "create-types": "tsc index.js --allowJs  --declaration --emitDeclarationOnly"
+        "test": "xo && nyc ava && tsc",
+        "create-types": "tsc -p declaration.tsconfig.json"
     },
     "files": [
         "index.js",
+        "index.d.ts",
+        "index.d.ts.map",
```

Comment:
I don't think it should publish a Source Map

Reply:
Why not? It's just a way to ensure that "go to source" in eg. VSCode actually still goes to the source and not to the type file?

And its only this big:

```json
{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["index.js"],"names":[],"mappings":"AA+BA,0CAHW;IAAE,KAAK,CAAC,EAAE,OAAO,GAAG,SAAS,CAAA;CAAE,GAC7B,OAAO,CAgFnB"}
```

Reply:
Related: https://github.com/Microsoft/TypeScript/issues/14479
And https://www.typescriptlang.org/docs/handbook/project-references.html#declarationmaps

But sure, I can remove it

Reply:
Removed ✔️